### PR TITLE
Release 22.3.0

### PR DESCRIPTION
**Pull request by jakirkham**
Includes a needed fix for `click` version `8.1.0+` ( https://github.com/psf/black/issues/2964 )

<hr>

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by vyasr** [comment]
Thanks for opening this John!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,7 +19,6 @@ outputs:
       noarch: python
       entry_points:
         - black = black:patched_main
-        - black-primer = black_primer.cli:main
```
Comment (jakirkham) [inline recipe/meta.yaml:22]: `black-primer` was dropped upstream in PR ( https://github.com/psf/black/pull/2924 )

**Comment by jakirkham** [comment]
This is ready for review @conda-forge/black 😄

**Comment by carlodri** [comment]
Thanks @jakirkham!

**Comment by jakirkham** [comment]
Thanks Carlo! 😄

**Pull request merged**